src/plugins/displayLayout/components/DisplayLayout.vue

```diff
@@ -152,29 +152,26 @@ export default {
         }
     },
     data() {
-        let domainObject = JSON.parse(JSON.stringify(this.domainObject));
-
         return {
-            internalDomainObject: domainObject,
             initSelectIndex: undefined,
             selection: [],
             showGrid: true
         };
     },
     computed: {
         gridSize() {
-            return this.internalDomainObject.configuration.layoutGrid;
+            return this.domainObject.configuration.layoutGrid;
```

Comment:
No longer necessary to track two different instances of an object (eg. `internalDomainObject` or `observedDomainObject` vs . just `domainObject`).